internal/handlers/allMetricsHandler.go

```diff
@@ -0,0 +1,25 @@
+package handlers
+
+import (
+	"fmt"
+	"github.com/sinobite/go-metrics/internal/storage"
+	"io"
+	"net/http"
+	"strings"
+)
+
+func AllMetricsHandler(writer http.ResponseWriter, request *http.Request) {
+	var metricsSlice []string
+	for key, value := range storage.Storage.Gauges {
+		metricsSlice = append(metricsSlice, fmt.Sprintf("%s: %v \n", key, value))
+	}
+	for key, value := range storage.Storage.Counters {
+		metricsSlice = append(metricsSlice, fmt.Sprintf("%s: %v \n", key, value))
+	}
+
+	writer.Header().Set("Content-Type", "text/html")
+	_, err := io.WriteString(writer, strings.Join(metricsSlice, ""))
+	if err != nil {
+		panic(err)
```

Comment:
Не стоит паниковать `panic(err)` в хендлере, так как это приведет к крашу приложения. Лучше обработать ошибку, залогировать ее и попробовать вернуть сообщение об ошибке:

```suggestion
		http.Error(writer, "Failed to write metrics", http.StatusInternalServerError)
		log.Printf("Failed to write metrics: %v", err) 
```